Network/Nmcli.hs

```diff
@@ -27,7 +27,7 @@ scanCmd = Scan "nmcli --terse --fields ssid,signal dev wifi"
 
 -- | Command to list the wifi the computer can currently auto connect to
 knownCmd :: Command
-knownCmd = Scan "nmcli --terse --fields name con list"
```

Comment:
You can check on your side (without merging) if it's working with this:

```sh
nmcli --terse --fields name con
```
vs the original:

```sh
nmcli --terse --fields name con list
```